### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by dormancygrace**
Add volume for redis, use latest point releases of redis and memcached

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ services:
     image: postgres:${POSTGRES_VERSION}
     restart: ${RESTART}
     volumes:
+      - postgresql-data:/var/lib/postgresql/data
```
Comment (monotek) [inline docker-compose.yml:21]: thats not needed as the backup works over a network connection

**Review by monotek — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -82,8 +83,10 @@ services:
       - zammad-data:/opt/zammad
 
   zammad-redis:
-    image: redis:7.0.5-alpine
+    image: redis:7.0-alpine
     restart: ${RESTART}
+    volumes:
```
[the remark this answers is not in the document]
Comment (dormancygrace) [reply docker-compose.yml:88]: To prevent creation of volumes

File: docker-compose.yml (hunk 1 above)
Comment (dormancygrace) [reply docker-compose.yml:21]: Same as redis volume, but in fact adding this volume breaks backup (

File: docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (monotek) [reply docker-compose.yml:88]: But your change creates one?

**Comment by monotek** [comment]
redis volume will be added in: https://github.com/zammad/zammad-docker-compose/pull/347

**Comment by monotek** [comment]
redis volume will be added in: https://github.com/zammad/zammad-docker-compose/pull/347

**Pull request closed without merging**